### DISCUSSION TITLE
How to add image from drawable rather than gallery?

### DISCUSSION THREAD

**Discussion by BaniHani** [burhanrashid52/PhotoEditor#321] [Q&A] [answered]

**Comment by basitis** [comment]
@BaniHani Do you mean to add as a main image or as sticker image?

**Comment by BaniHani** [comment]
as a main image please.

**Comment by basitis** [comment, marked as answer]
@BaniHani If you have fix drawable then you can directly give to the xml and if you want to load runtime then library is expose "source"(ImageView).  You can access something like this:

`
photoFrameEditorView.source.setImageDrawable(getResources().getDrawable(R.drawable.my_drawable));`

**Comment by basitis** [comment]
@BaniHani If your issue is resolved with my comment then please close this issue. Thanks in advance.
